package.json

```diff
@@ -48,7 +48,7 @@
     "karma-phantomjs-launcher": "^0.2.0",
     "karma-phantomjs-shim": "^1.0.0",
     "karma-sinon-chai": "^1.0.0",
-    "karma-webpack": "^1.5.1",
+    "karma-webpack": "1.5.1",
```

Comment:
Yea you're right, I thought that a fresh `npm install` would be installing karma 0.13.3 but I forgot that's not how carets work for 0.x.x is it!